src/scripts/heroAnimation/hero.js

```diff
@@ -0,0 +1,256 @@
+let width,
+  height,
+  ctx,
+  points,
+  target,
+  canvasTop,
+  animateHeader = true,
+  timer = null;
+const canvas = document.getElementById("js-canvas-hero");
+const container = document.querySelector(".grav-c-hero");
+
+function HeroAnimation() {
+  if (container !== null) {
+    addListeners();
+    initHeader();
+    initAnimation();
+  }
+}
+
+function resetTimer() {
+  window.clearTimeout(timer);
+  timer = null;
+  timer = window.setTimeout(initHeader, 60000);
+}
+
+function scrollCheck() {
+  if (document.body.scrollTop > height) {
+    animateHeader = false;
+  } else {
+    animateHeader = true;
+  }
+}
+
+function resizeCanvas() {
+  canvasTop = canvas.getBoundingClientRect().top;
+  canvas.setAttribute(
```

Comment:
What is the purpose of this?

CSS is resizing the canvas, then the JS measures the new size of the canvas and explicitly sets the width and height. That feels redundant.

If this is intentional, then perhaps add a comment to explain the reason.

Reply:
Not sure why that was there, it was in the original but it's all refactored now.